flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/types/TaskManagerSlot.java

```diff
@@ -139,6 +139,16 @@ public boolean isMatchingRequirement(ResourceProfile required) {
 		return resourceProfile.isMatching(required);
 	}
 
+	@Override
+	public String toString() {
+		return "TaskManagerSlot{" +
```

Comment:
`resourceProfile` and `taskManagerConnection` are deliberately not included?

Reply:
Will add them.